closed/make/CopyFiles.gmk

```diff
@@ -37,6 +37,31 @@ ifeq ($(OPENJDK_TARGET_OS), windows)
   COPY_FILES += $(MSVCP_TARGET)
 endif
 
+ifneq ($(OPENSSL_BUNDLE_LIB_PATH), )
+  ifeq (true,$(OPENJ9_ENABLE_JITSERVER))
+    # Copy OpenSSL SSL Lbrary
+    # To bundle first search for libssl 1.1.x library, if not found, search for 1.0.x
+    ifeq ($(OPENJDK_TARGET_OS), linux)
+      OPENSSL_SSL_LIB_NAME = libssl.so.1.1
+      ifeq ("$(wildcard $(OPENSSL_BUNDLE_LIB_PATH)/$(OPENSSL_SSL_LIB_NAME))", "")
+        OPENSSL_SSL_LIB_NAME = libssl.so.1.0.0
+        ifeq ("$(wildcard $(OPENSSL_BUNDLE_LIB_PATH)/$(OPENSSL_SSL_LIB_NAME))", "")
+          OPENSSL_SSL_LIB_NAME =
+        endif
+      endif
+    endif # OPENJDK_TARGET_OS
+
+    ifneq ($(OPENSSL_SSL_LIB_NAME), )
+      OPENSSL_SSL_TARGET_LIB = $(JDK_OUTPUTDIR)/lib$(OPENJDK_TARGET_CPU_LIBDIR)/$(OPENSSL_SSL_LIB_NAME)
+
+      $(OPENSSL_SSL_TARGET_LIB): $(OPENSSL_BUNDLE_LIB_PATH)/$(OPENSSL_SSL_LIB_NAME)
+		$(CP) $(OPENSSL_BUNDLE_LIB_PATH)/$(OPENSSL_SSL_LIB_NAME) $@
+
+      COPY_FILES += $(OPENSSL_SSL_TARGET_LIB)
+    endif # OPENSSL_SSL_LIB_NAME
+  endif # OPENJ9_ENABLE_JITSERVER
+endif # OPENSSL_BUNDLE_LIB_PATH
```

Comment:
[1] copies `libcrypto` not `libssl`.  This part of the code makes sure `libssl` is bundled. 

[1] https://github.com/ibmruntimes/openj9-openjdk-jdk8/blob/openj9/jdk/make/CopyFiles.gmk#L252-L326

Reply:
Ok, I missed that difference; I think the part existing code that copies `libcypto` should also copy `libssl` rather than have the copying in two places.